src/index.js

```diff
@@ -89,6 +89,14 @@ export default function commonjs ( options = {} ) {
 						const match = exportsPattern.exec( flattened.keypath );
 						if ( !match || flattened.keypath === 'exports' ) return;
 
+						if ( flattened.keypath === 'module.exports' && node.right.type === 'ObjectExpression' ) {
+							return node.right.properties.forEach( prop => {
+								if ( prop.computed ) return;
+								const name = prop.key.name;
+								if ( name === makeLegalIdentifier( name ) ) namedExports[ name ] = true;
```

Comment:
`prop.key.name` is only valid for `Identifier`s, all of which should be valid, right? If so, it should suffice to do:

```js
if ( prop.computed || prop.key.type !== 'Identifier' ) return;
namedExports[ prop.key.name ] = true;
```